### DISCUSSION TITLE
Add default ordering in models

### DISCUSSION THREAD

**Discussion by nikosmichas** [revsys/django-friendship#157] [Ideas]
Hello!
It seems that the app models don't have a default order.
I was wondering if it would make sense to add ordering by id as a default in order to have a consistent response when using eg `Friend.objects.requests(user)` in a serializer with pagination.
What do you think?

The only change needed would be adding this to all models

```python
class Meta:
   ordering = ['id']
```

source: https://docs.djangoproject.com/en/3.2/ref/models/options/#ordering

I would happily open a PR if you agree with the approach

**Comment by frankwiles** [comment]
Actually I prefer not to have default ordering on models 99% of the time.  The reason is you do pay a small performance penalty for the `ORDER BY` even when you aren't using it.  For example, if I have a cron job that needs to do query a bunch of stuff but order doesn't matter.  

I find it much better to just remember to order the querysets on the views when I need to do things paginated or otherwise need order.
